src/Core/src/ViewExtensions.cs

```diff
@@ -98,12 +98,7 @@ internal static bool NeedsContainer(this IView? view)
 				return true;
 #endif
 
-#if ANDROID || IOS
-#pragma warning disable CS0618 // Type or member is obsolete
-			if (view is IBorder border && border.Border != null)
-				return true;
-#pragma warning restore CS0618 // Type or member is obsolete
-#elif WINDOWS
+#if WINDOWS || ANDROID
```

Comment:
The UITest CollectionViewMeasureFirstItem is failing on Android after the changes. The snapshot have small differences. The sample use a CV with Borders in the DataTemplate, could you take a look?
Let me know if can help.

Reply:
I hope it will work now

Reply:
Hmm they seem to be identical to me. What do you think?

|Main|PR|
|--|--|
|<img src="https://github.com/user-attachments/assets/eecfdd1c-2306-4dca-bef3-9ec7a98d37b8" width="300px"/>|<img src="https://github.com/user-attachments/assets/10ad7ca6-a017-437f-9fff-54e205870c17" width="300px"/>|

But the test shows that they differ somehow...
<img src="https://github.com/user-attachments/assets/a6bde10c-2b42-4ca0-9259-7a052d77041f" width="300px"/>